### DISCUSSION TITLE
Add FL Studio 20 to "Projects using svgren" list

### DISCUSSION THREAD

**Discussion by SuganthanBC** [igagis/svgren#102] [General]
I have just noticed that FL Studio 20 uses this library. Perhaps, it can be added to [Projects using svgren](https://github.com/cppfw/svgren/blob/master/wiki/usages.adoc)?

**Comment by igagis** [comment]
Well, sure, if that's the case. How did you know that? Is it mentioned anywhere?
I know that one guy working for FL was interested in this lib, but I was not sure if he is actually using it in the final product...

**Comment by SuganthanBC** [comment]
Sure, here it is. I'm unsure how much it uses though. But still...

https://user-images.githubusercontent.com/5999762/152349943-ca28ac55-809c-4631-996e-925c48e72d38.mp4

**Reply by igagis** [reply]
@SuganthanBC 
Thanks, I have added it to the list! Nice to know that this popular software uses my lib :). Thanks for finding this!
